pkg/cluster/k8sres.go

```diff
@@ -960,12 +977,14 @@ func (c *Cluster) getPodEnvironmentConfigMapVariables() ([]v1.EnvVar, error) {
 				metav1.GetOptions{})
 		}
 		if err != nil {
-			return nil, fmt.Errorf("could not read PodEnvironmentConfigMap: %v", err)
+			return configMapPodEnvVarsList, fmt.Errorf("could not read PodEnvironmentConfigMap: %v", err)
```

Comment:
return empty list instead of nil so that appending works nicely in generateSpiloEnvVars